pywhy_stats/_version.py

```diff
@@ -0,0 +1,5 @@
+"""Version number."""
+
+from importlib.metadata import version  # type: ignore
```

Comment:
Do we really need this? I guess this causes issues with older versions, but I also don't see the benefit of it?

Reply:
Honestly... not sure how this works exactly. @darthtrevino added this in dodiscover. My impression is that it allows us to only have version in one place: `pyproject.toml` which makes life easier in terms of making releases(?)